src/command_modules/azure-cli-batch/azure/cli/command_modules/batch/_validators.py

```diff
@@ -208,7 +185,7 @@ def validate_cert_file(namespace):
         raise ValueError("Cannot access certificate file: " + namespace.certificate_file)
 
 
-def validate_options(namespace):
+def validate_options(cmd, namespace):
```

Comment:
You don't have to add `cmd` if you don't use it. You might need a pylint statement here to stop it from complaining.